src/utils/utils.ts

```diff
@@ -45,53 +48,57 @@ export default class Utilities {
         return true
     }
 
-    static getSatelitesFromFile(): Array<string[]> {
-        const data = fs.readFileSync('./ucs-satelite-db.txt', 'utf8')
+    static getSatelitesFromFile(): arrayOfSatellites {
+        const data = fs.readFileSync('./satellites-db.json', 'utf8')
         if (!data) return []
-        const satelites = data.split('\n').slice(1)
-        const usableDataMaxIndex = this.satelliteHeaders.length
-        const splittedSatelitesData: Array<string[]> = []
-        satelites.forEach((value) => { splittedSatelitesData.push(value.split('\t').slice(0, usableDataMaxIndex)) })
-        return splittedSatelitesData.slice(0, splittedSatelitesData.length - 1)
+        return JSON.parse(data)
     }
 
     static countSatelitesFromFile(): number {
-        const data = fs.readFileSync('./ucs-satelite-db.txt', 'utf8')
-        if (!data) return 0
-        return data.split('\n').length - 2
+        return this.getSatelitesFromFile().length as number
     }
 
-    static prePopulateDatabase(satellites: SatelliteInterface[]): Promise<SatelliteInterface[]> {
-        const satellitesCreation: Promise<SatelliteInterface>[] = []
-        satellites.forEach((satellite) => {
-            satellitesCreation.push(SatelliteUtilities.createSatelite(satellite as SatelliteInterface))
+    static standardizeSatellitesData(satellites: arrayOfSatellites): SatelliteType[] {
+
+        const standarizedSatellites: arrayOfSatellites = satellites.map((item) => {
```

Comment:
`standarizedSatellites` isn't really normalized until this function returns .map which only then returns normalized satellites, so this name can be confusing. I suggest renaming it to `satellitesToNormalize`, or maybe even better, removing this variable and instead chaining both .map's together like
```ts
return satellites.map(fillMissingKeys).map(parseNumbers)
```